storm/src/main/storm/mesos/MesosNimbus.java

```diff
@@ -107,13 +111,24 @@
   public static final String CONF_MESOS_CONTAINER_DOCKER_IMAGE = "mesos.container.docker.image";
   public static final String CONF_MESOS_SUPERVISOR_STORM_LOCAL_DIR = "mesos.supervisor.storm.local.dir";
   public static final String FRAMEWORK_ID = "FRAMEWORK_ID";
+  public static final String DEFAULT_MESOS_COMPONENT_NAME_DELIMITER = "|";
+
+  public static final String CONF_ZOOKEEPER_SERVERS = "storm.zookeeper.servers";
+  public static final String CONF_ZOOKEEPER_PORT = "storm.zookeeper.port";
+  public static final String CONF_STORM_LOGVIEWER_ZK_DIR = "storm.logviewer.zookeeper.dir";
```

Comment:
Here's another thought: right now there are basically 3 different dirs where state is stored related to StaaS:
1. `/<storm-something>/`:  this is where Storm itself stores some state.  It is configured by `storm.zookeeper.root` in `storm.yaml`.
2. `/<mesos-something>/`: this is where Mesos stores some state.
3. `/consumers/`: this is where we tell our users to store their Kafka consumer state (when they are using ZK).

So we're creating a 4th place.  My question is:  would it be beneficial to have a 4. `/<storm-mesos-something>` specific dir, and then have the logviewer dir be a subdir under that?   e.g., I can see us putting other kinds of state in here eventually.